### PR TITLE
feat(docs): nav pg

### PR DESCRIPTION
**Pull request by jaketrent**
sdf

### CODE REVIEW

File: docs/src/pages/components/nav.mdx [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+    <NavItem icon={<HomeIcon />} menu={<div>placeholder</div>}>Label</NavItem>
+    <NavItem icon={<HomeIcon />}>Label</NavItem>
+    <NavItem menu={<div>placeholder</div>}>Label</NavItem>
+    <NavItem>Label</NavItem>
+    <NavItem icon={<HomeIcon />} alignment={NavItem.alignments.vertical}>Label</NavItem>
+    <NavItem icon={<HomeIcon />} alignment={NavItem.alignments.vertical} menu={<div>placeholder</div>}>Label</NavItem>
+    <NavItem icon={<HomeIcon />} />
+  </div>
+}
+
```
Comment (danethurber) [inline docs/src/pages/components/nav.mdx:101]: this example is not visible in light theme

File: docs/src/pages/components/nav.mdx [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+function Example() {
+  return <NavBrand logo={<MyLogo />} wordmark="TITLE" />
+}
+
+export default Example
+
+function MyLogo() {
+  return (
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (danethurber) [inline docs/src/pages/components/nav.mdx:147]: this logo isn't visible to me

File: docs/src/pages/components/nav.mdx [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+          <NavItem icon={<AccountIcon />} />
+        </div>
+      </React.Fragment>
+    }
+  />
+}
+
+export default Example
+
+function MyLogo() {
```
Comment (danethurber) [inline docs/src/pages/components/nav.mdx:326]: also not visible

**Review by danethurber — CHANGES_REQUESTED** [review]

**Comment by jaketrent** [comment]
Hey Dane, thanks for the look.  Yeah, the nav elements are not light-theme supported.  They're only meant to appear in the navbar, with a dark background. I solved that in the examples, perhaps with a bandaid. I don't love the visual result. Not sure what might be best.

**Review by danethurber — APPROVED** [review]
bandaid seems ok to me. these are the only components that work this way 👍

**Pull request merged**